test/import/FSHImporter.Profile.test.ts

```diff
@@ -0,0 +1,528 @@
+import {
+  assertCardRule,
+  assertFixedValueRule,
+  assertFlagRule,
+  assertOnlyRule,
+  assertValueSetRule
+} from '../utils/asserts';
+import { importText } from '../../src/import';
+import { Code, Quantity, Ratio } from '../../src/fshtypes';
+
+describe('FSHImporter', () => {
+  describe('Profile', () => {
+    describe('#sdMetadata', () => {
+      it('should parse the simplest possible profile', () => {
+        const input = `
+        Profile: ObservationProfile
+        Parent: Observation
+        `;
+
+        const result = importText(input);
+        expect(result.profiles.size).toBe(1);
+        const profile = result.profiles.get('ObservationProfile');
+        expect(profile.name).toBe('ObservationProfile');
+        expect(profile.parent).toBe('Observation');
+        // if no id is explicitly set, should default to name
+        expect(profile.id).toBe('ObservationProfile');
+      });
+
+      it('should parse profile with additional metadata properties', () => {
+        const input = `
+        Profile: ObservationProfile
+        Parent: Observation
+        Id: observation-profile
+        Title: "An Observation Profile"
+        Description: "A profile on Observation"
```

Comment:
Do we want to test multi-line descriptions as well? I think that'll be a common use case, so we probably want a test to validate it.

Reply:
I do test it in one spot (the `FixedValueRule` tests, I think) -- but I can test it here too.